src/peft/import_utils.py

```diff
@@ -63,3 +63,19 @@ def is_torch_tpu_available(check_device=True):
                 return False
         return True
     return False
+
+
+def is_auto_awq_available():
+    if importlib.util.find_spec("awq") is not None:
+        # TODO: change it to 0.2.0
```

Comment:
This is not quite clear to me: change it when?

Reply:
Sorry I'll remove it